android/fastlane/Fastfile

```diff
@@ -76,7 +76,7 @@ platform :android do
     upload_to_browserstack_app_automate(
       browserstack_username: ENV["BROWSERSTACK_USER"],
       browserstack_access_key: ENV["BROWSERSTACK_ACCESS_KEY"],
-      file_path: "./app/build/outputs/apk/release/app-universal-release.apk"
+      file_path: ENV["GRADLE_APK_OUTPUT_PATH"]
```

Comment:
I think I had set this before but `GRADLE_APK_OUTPUT_PATH` was not consistently returning the universal build, fyi. maybe I'm mistaking but something to keep in mind.

Reply:
Testing this it seems to work